guides/plugins/apps/administration/add-custom-modules.md

```diff
@@ -0,0 +1,150 @@
+# Add custom modules
+
+## Overview
+
+In your app, you are able to add your own modules to the administration. Your custom modules are loaded as iframes which are embedded in the Shopware administration and within this iframe, your website will be loaded and shown.
+
+
+Creating custom modules takes place at the `<admin>` section of your `manifest.xml`.
+Take a look at the [Manifest Reference](../../../../resources/references/app-reference/manifest-reference.md)
+You can add anny amount of custom modules by adding new `<module>` elements to your manifest.
+
+To configure your module you can set it up with with some addtional attributes.
+* `name` (required): The technical name of the module. This is the name your module is referenced with.
+* `source` (optional): The URL to your app servers endpoint from which the module is served from. This can be omitted if you want to define a menu item that should serve as a parent menu item for other app modules.
+* `parent` (optional): The administration navigation id from the menu item that serves as the parrent menu item. If omitted your module will be listed under the "My apps" menu entry. **This field will be required in future versions as we are going to remove the My Apps menu item**
+* `position` (optional): A numeric index that sets the position of your menu entry regarding to it's sibblings.
+
+Additionally you can define `label` elements inside of your `module` element, to set up how your module will be displayed in the admin menu.
+
+{% code title="manifest.xml" %}
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/master/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd">
+    <meta>
+        ...
+    </meta>
+    <admin>
+        <module name="exampleModule"
+                source="https://example.com/promotion/view/promotion-module"
+                parent="sw-marketing"
+                position="50"
+        >
+            <label>Example module</label>
+            <label lang="de-DE">Beispiel Modul</label>
+        </module>
+    </admin>
+</manifest>
+
+```
+{% endcode %}
+
+For a complete reference of the structure of the manifest file take a look at the [Manifest reference](../../../../resources/references/app-reference/manifest-reference.md).
+
+
+
+If the user opens the module in the administration your app will receive a request to the URL defined in the `source` attribute of your `module` element. Your app can determine the shop that has opened the module through query parameters added to the url: 
+
+* shop-id: The unique identifier of the shop, where the app was installed
```

Comment:
Would you like to use `code` syntax here as well? Same as you did in a list before. 🙂